cpp/arcticdb/python/python_handlers.cpp

```diff
@@ -0,0 +1,24 @@
+#include <python/python_handlers.hpp>
+#include <python/gil_lock.hpp>
+
+namespace arcticdb {
+    void EmptyHandler::handle_type(
+            const uint8_t*& data ARCTICDB_UNUSED,
+            uint8_t* dest,
+            size_t dest_bytes,
+            const VariantField& encoded_field_info ARCTICDB_UNUSED,
+            const entity::TypeDescriptor& type_descriptor ARCTICDB_UNUSED,
+            std::shared_ptr<BufferHolder> buffers ARCTICDB_UNUSED
+    ) {
+        ARCTICDB_SAMPLE(HandleEmpty, 0)
+        util::check(dest != nullptr, "Got null destination pointer");
+        py::none none = {};
+        const size_t num_rows = dest_bytes / get_type_size(DataType::EMPTYVAL);
+        const PyObject** ptr_dest = reinterpret_cast<const PyObject**>(dest);
```

Comment:
There is an assumption here that `sizeof(PyObject**) == 8`, so we should at least `internal::check` that

Reply:
Do it at compile time?

Reply:
It's ratther `sizeof(PyObject*) == 8` I'll add a `static_assert`